### PR TITLE
Adds handholding

### PR DESCRIPTION
**Pull request by kevinz000**
[(M.zone_sel.selecting == "l_hand"||M.zone_sel.selecting == "r_hand")?("by [gender==FEMALE?("her"):(gender==MALE?"his":"their")] hands"):("passively")]

### CODE REVIEW

**Comment by Pacmandevil** [comment]
Begone, Sexual deviant.

**Comment by BurgerLUA** [comment]
would be better if some of the language was changed so it's nicer during erp

don't want a rude "YOU GRABBED HER HANDS PASSIVELY!!!" ruining the mood

**Comment by Hatterhat** [comment]
@kevinz000 buddy you were supposed to keep this bullshit to /tg/station what the fuck are you doing

**Comment by kevinz000** [comment]
@Hatterhat : ^ )

**Comment by ToasterStrudes** [comment]
How will this heresy affect gender-less things like Baseline and Industrial IPCs?

**Comment by kevinz000** [comment]
Done

**Comment by BurgerLUA** [comment]
How will this heresy affect tentacle hands like Dionaea?

File: code/modules/mob/living/carbon/human/human_attackhand.dm [hunk 1]
```diff
@@ -167,7 +167,7 @@
 				G.icon_state = "grabbed1"
 				visible_message("<span class='warning'>[M] gets a strong grip on [src]!</span>")
 				return 1
-			visible_message("<span class='warning'>[M] has grabbed [src] passively!</span>")
+			visible_message("<span class='warning'>[M] has grabbed [src] [(M.zone_sel.selecting == "l_hand"||M.zone_sel.selecting == "r_hand")?("by [gender==FEMALE?("her"):(gender==MALE?"his":"their")] hands"):("passively")]!</span>")
```
Comment (Lohikar) [inline code/modules/mob/living/carbon/human/human_attackhand.dm]: I feel like this could be done simpler using BYOND gender macros.

**Comment by JackBewt** [comment]
This violates the lore. Loremaster veto, veto, veto!!

**Comment by skull132** [comment]
lol.

**Pull request closed without merging**